src/components/Task/AddRemove/AddRemoveTask.tsx

```diff
@@ -65,14 +46,14 @@ export const AddRemoveTask = ({ index }: AddRemoveTaskProps): JSX.Element => {
 		}));
 		modifiedTaskArr.splice(ind + IND_INC, MODIFIED_TASK_SPLICE_DELETE_COUNT, {
 			autoDueDate: false,
-			color: fetchRandomColor(),
-			document: "Uh Oh",
-			dueDate: new Date(),
+			color: tasks[index].color,
+			document: "Type Document Text Here.",
+			dueDate: tasks[index].dueDate,
 			id: modifiedTaskArr.length + CONSTANTS.TASK_INDEX_INC,
-			name: "Swag",
+			name: "New Task",
```

Comment:
Thank you, when I made the presentation video I got jumpscared by "Swag" and "Uh Oh"